core/src/main/java/com/adobe/aem/guides/wknd/core/dto/Results.java

```diff
@@ -0,0 +1,127 @@
+package com.adobe.aem.guides.wknd.core.dto;
+
+import java.util.ArrayList;
+
+public class Results {
+    public String title;
+    public String link;
+    public ArrayList<String> keywords;
+    public ArrayList<String> creator;
+    public Object video_url;
+    public String description;
+    public String content;
+    public String pubDate;
+    public String image_url;
```

Comment:
Use camel case instead of underscore as and when needed. Refer attached points for the advantages-

Camel casing has a larger probability of correctness than underscores. (odds are 51.5% higher)
On average, camel case took 0.42 seconds longer, which is 13.5% longer.
Training has no statistically significant impact on how style influences correctness.
Those with more training were quicker on identifiers in the camel case style.
Training in one style, negatively impacts the find time for other styles.